config/crd/bases/multicluster.x-k8s.io_clusterprofiles.yaml

```diff
@@ -0,0 +1,176 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.14.0
+  name: clusterprofiles.multicluster.x-k8s.io
+spec:
+  group: multicluster.x-k8s.io
+  names:
+    kind: ClusterProfile
+    listKind: ClusterProfileList
+    plural: clusterprofiles
+    singular: clusterprofile
+  scope: Cluster
```

Comment:
is this our final decision?

Reply:
No, this is open for discussion. This PR is meant to drive some of these discussions in async manner.
We feel like Cluster scope is a more natural fit for this API so it's a good starting point.
CC @deads2k

Reply:
My general guidance for creating new CRDs is that "if the CRD doesn't have to be cluster-scoped, it should be namespace-scoped."
Also, if we had a namespace with a predefined name for all multi-cluster management objects, such as cluster profile, and other future objects, creating RBAC rules for accessing them in that namespace would be simpler.

Reply:
I don't have a _strong_ feeling, mostly because we just don't know what we don't know, and need to explore it.

That said, my instinct is that if you CAN namespace it, you probably should, unless there's a good reason not to.  We have plenty of examples of cluster-scoped things we wish we had namespaced, and few to none of the reverse examples.

Reply:
Discussion is happening here: https://docs.google.com/document/d/1E_z3ti-d-modwnhsvR3yBZwX4rRpL26dKkl41YAptRo/edit

Reply:
KEP PR to clarify the scope of ClusterProfile: https://github.com/kubernetes/enhancements/pull/4620
This CRD implementation will follow the KEP once it's merged.

Reply:
thanks for the update @mikeshng

Reply:
Updated to namespaced scope. PTAL.